impl/initiating_test.go

```diff
@@ -85,75 +85,6 @@ func TestDataTransferInitiating(t *testing.T) {
 				testutil.AssertFakeDTVoucher(t, receivedRequest, h.voucher)
 			},
 		},
-		"Remove Timed-out request": {
```

Comment:
These tests were for connection monitoring, which has now been removed in favour of the push / pull channel monitors